packages/api/src/queue-processor.ts

```diff
@@ -132,6 +160,21 @@ const main = async () => {
     }
   )
 
+  const queue = await getBackendQueue()
+  if (queue) {
+    await queue.add(
```

Comment:
i think we might want to use `jobId` here to deduplicate jobs

Reply:
if we have ten instances, will it create ten jobs?

Reply:
I _think_ bullmq handles this for us. It generates a unique jobid for each run using the job name. I think beca